ketcher-autotests/tests/User-Interface/Toolbar/toolbar-ui.spec.ts

```diff
@@ -0,0 +1,81 @@
+import { test } from '@playwright/test';
+import {
+  takeEditorScreenshot,
+  takeLeftToolbarScreenshot,
+  takeTopToolbarScreenshot,
+  FunctionalGroups,
+  clickInTheMiddleOfTheScreen,
+  selectFunctionalGroups,
+  pressButton,
+  STRUCTURE_LIBRARY_BUTTON_NAME,
+  DELAY_IN_SECONDS,
+  delay,
+  resetCurrentTool,
+  selectLeftPanelButton,
+  LeftPanelButton,
+} from '@utils';
+
+test.describe('Open Ketcher', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto('');
+  });
+
+  test('Toolbar palette: full screen verification', async ({ page }) => {
+    /* 
+      Test case: EPMLSOPKET-1331
+      Description:  Toolbar - Toolbar palette: full screen verification
+      */
+    await takeLeftToolbarScreenshot(page);
+  });
+
+  test('Help: UI Verification', async ({ page }) => {
+    /*
+        Test case: Test case: EPMLSOPKET-1328
+        Description: Help button tooltip verification
+        */
+    await page.getByTitle('Help (?)').first().hover();
```

Comment:
JFYI, "tooltip" is shown by browser, not by Ketcher. And the tooltip itself just depends on "title" attribute of DOM-node. https://www.w3schools.com/tags/att_global_title.asp#:~:text=HTML%20title%20Attribute&text=The%20title%20attribute%20specifies%20extra,validate%20on%20any%20HTML%20element.

Actually, you can simplify the test (and future tests) just by (example code, may not work):
```
const helpButton = page.getByTestId('help-button');
await expect(someButton).toHaveAttribute('title', 'Help (?)');
```

You can refer: https://playwright.dev/docs/api/class-locatorassertions#locator-assertions-to-have-attribute